app/discourse/models.py

```diff
@@ -30,6 +30,10 @@ class DiscourseTopicEvent(models.UserEvent):
 
     posts_count = Column(types.Integer)
 
+    category_name = Column(types.Text)
+
+    category_slug = Column(types.Text)
```

Comment:
This data is denormalized but that's fine for now because `DiscourseTopicEvent` is just a cache really.

Once we add privacy controls and start showing users restricted stuff, though, we might want to create a separate table for categories. We'll cross that bridge when we get there.